packages/block-editor/package.json

```diff
@@ -23,15 +23,31 @@
 	"react-native": "src/index",
 	"dependencies": {
```

Comment:
- `block-editor/src/components/block-compare/index.js` includes an import from `diff`
- `block-editor/src/components/block-list/block-html.js` includes an import from `react-autosize-textarea`

`react-autosize-textarea` is now used in both `editor` and `block-editor`. Should we be concerned about its size impact?

Reply:
https://bundlephobia.com/result?p=react-autosize-textarea@3.0.2

It's smallish but yeah it feels like we maybe be able to extract a reusable component and use it in all the modules.